AppController/djinn.rb

```diff
@@ -2834,6 +2834,17 @@ def get_all_appengine_nodes()
     return ae_nodes
   end
 
+  # Gets a list of autoscaled nodes by going through the nodes array
+  # and splitting the array from index greater than the 
+  # minimum images specified.
+  def get_autoscaled_nodes()
+    autoscaled_nodes = []
+    min_images = Integer(@options['min_images'])
+    @state_change_lock.synchronize {
+      autoscaled_nodes = @nodes.drop(min_images)
```

Comment:
So we believe that the @nodes array does maintain the order of the added nodes because we do some additional operations which depends on the order. But yes, we should have a better indicator of whether or not the node is autoscaled and I can work on it in a separate PR.